compiler/rustc_mir_transform/src/coroutine.rs

```diff
@@ -1293,12 +1287,12 @@ fn create_coroutine_resume_function<'tcx>(
                         kind: TerminatorKind::Goto { target: poison_block },
                     };
                 }
-            } else if !block.is_cleanup {
+            } else if !block.is_cleanup &&
```

Comment:
Formatting dictates that `&&` should be leading, not trailing. Move this to the beginning of the next `let`.